resources/config

```diff
@@ -23,6 +23,9 @@
     //         "default": ""
     //     }
     // },
+    "sway/mode": {
```

Comment:
it's not really useful since the default format is {} :?

Reply:
I'd be happy to remove this. I have only included this to let people know `format` is available.